tests/test_interface/test_text_interface.py

```diff
@@ -55,6 +55,14 @@ def test_rotation(self):
         )
         assert text._create_qt_object().rotation() == 123
 
+    #   def test_rotation_about_transform_origin(self):
+    #       text = TextInterface(ORIGIN, self.brush, self.pen, "foo", self.font)
+    #       assert text._create_qt_object().transform_origin() == ORIGIN
+    #       text = TextInterface(
+    #           ORIGIN, self.brush, self.pen, "foo", self.font, rotation=(Unit(12),Unit(12))
+    #       )
+    #       assert text._create_qt_object().transform_origin() == (Unit(12),Unit(12))
```

Comment:
Qt points are expressed as `QPointF` values. `neoscore.interface.qt.converters` has some convenience methods for converting neoscore points to Qt points.